docs/dev-machine-setup.md

```diff
@@ -183,6 +183,27 @@ set up a Ubuntu-based VM on GCE with nested KVM enablement can be found in GCE
 
   Now you can continue with the Firecracker [Getting Started](getting-started.md)
   instructions to install and configure Firecracker in the new VM.
+  
+  And also, to secure your VM. SetUp new Firewal rules and attach it as soon as possible.
```

Comment:
I believe this could be added in a subsection under GCP, called "GCP Security"/"Securing GCP" or something similar. I am suggesting this since lines 184-185 are ending the above steps by pointing out to `getting-started.md` and security aspects could get their own space . Also, I would rephrase like 187 as "Secure your VM by setting up new Firewall rules:" and insert the steps below.